docs/Using-EthSigner/Getting-Started.md

```diff
@@ -1,9 +1,13 @@
-description: Install EthSigner from binary distribution
+description: Getting started with EthSigner
 <!--- END of page meta data -->
 
 # Getting Started 
 
-EthSigner requires a V3 Keystore key file and a password file. 
+EthSigner requires a V3 Keystore key file and a password file for signing transactions
```

Comment:
Yeah - this is technically only for file-based signing

Reply:
Fixed